airflow/ui/src/mocks/handlers/log.ts

```diff
@@ -21,7 +21,7 @@
 import { http, HttpResponse, type HttpHandler } from "msw";
 
 export const handlers: Array<HttpHandler> = [
-  http.get("/public/dags/log_grouping/dagRuns/manual__2025-02-18T12:19/taskInstances/generate/-1", () =>
+  http.get("/api/v2/dags/log_grouping/dagRuns/manual__2025-02-18T12:19/taskInstances/generate/-1", () =>
```

Comment:
@pierrejeambrun Is it possible to configure the global http client with the `/api/v2` prefix so that each of these just becomes

```suggestion
  http.get("/dags/log_grouping/dagRuns/manual__2025-02-18T12:19/taskInstances/generate/-1", () =>
```

WDYT? Worth it?

Reply:
That's a good idea yes. I would suggest do that in a follow up PR though.

Reply:
@pierrejeambrun I'll let you open a tracking issue for this one, I'm not exactly sure what'd need to be done for it.